src/halmos/config.py

```diff
@@ -231,15 +243,22 @@ class Config:
     )
 
     array_lengths: str = arg(
-        help="set the length of dynamic-sized arrays including bytes and string (default: loop unrolling bound)",
+        help="specify lengths for dynamic-sized arrays, bytes, and string. Lengths can be specified as a comma-separated list of integers enclosed in curly braces, or as a single integer.",
```

Comment:
```suggestion
        help="specify lengths for dynamic-sized arrays, bytes, and string types. Lengths can be specified as a comma-separated list of integers enclosed in curly braces, or as a single integer.",
```

Reply:
does it really work with curly braces? in my shell it has a special meaning and does substitutions:

```sh
$ echo a={1,2}
a=1 a=2
```

Reply:
um, you can escape curly braces in various ways, e.g., `echo 'a={1,2}'`